Periodic_3_mesh_3/doc/Periodic_3_mesh_3/Periodic_3_mesh_3.txt

```diff
@@ -250,6 +250,8 @@ for further information.
 
 \section Periodic_3_mesh_3_section_interface Interface
 
+Since \cgal 5.6, this packages uses \ref bgl_namedparameters to set parameters. More details are provided in  \ref FromBoostNPtoCGALNP.
```

Comment:
``` suggestion
Since \cgal 5.6, this package uses \ref bgl_namedparameters to set parameters. More details are provided in \ref FromBoostNPtoCGALNP.
```